staging/src/k8s.io/kubectl/pkg/util/templates/normalizers_test.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package templates
+
+import (
+	"testing"
+)
+
+func TestLongDescMarkdown(t *testing.T) {
+	tests := []struct {
+		desc string
+		in   string
+		out  string
+	}{
+		{
+			desc: "Empty input produces empty output",
+			in:   "",
+			out:  "",
+		},
+		{
+			desc: "Single line text is preserved as is",
+			in:   "Some text",
+			out:  "Some text",
+		},
+		{
+			desc: "Consecutive new lines are combined into a single paragraph",
+			in:   "Line1\nLine2",
+			out:  "Line1 Line2",
+		},
+		{
+			desc: "Two paragraphs",
+			in:   "Line1\n\nLine2",
+			out:  "Line1\n\n Line2",
+		},
+		{
+			desc: "Leading and trailing spaces are stripped (single line)",
+			in:   "\t  \nThe text line  \n  \t",
+			out:  "The text line",
+		},
+		{
+			desc: "Leading and trailing spaces are stripped (multi line)",
+			in:   "\t  \nLine1\nLine2  \n  \t",
+			out:  "Line1 Line2",
+		},
+		{
+			desc: "List Items with order",
+			in:   "Title\n\n1. First item\n2. Second item\n\nSome text",
+			out:  "Title\n\n  1.  First item\n  2.  Second item\n\n Some text",
+		},
+		{
+			desc: "Multi lines without order",
+			in:   "\t\t\t\t\tDescriptions.\n\n * Item.\n * Item2.",
+			out:  "Descriptions.\n\n  *  Item.\n  *  Item2.",
+		},
```

Comment:
this unit test doesn't pass on `master` prior to the dependency bump:

```
--- FAIL: TestLongDescMarkdown (0.00s)
    --- FAIL: TestLongDescMarkdown/Multi_lines_without_order (0.00s)
        normalizers_test.go:75: expected(37):
            Descriptions.
            
              *  Item.
              *  Item2.
            =====
            got(39):
            Descriptions.
              
              *  Item.
              *  Item2.
```

Reply:
```
@@ -58,12 +58,8 @@ func (r *ASCIIRenderer) RenderNode(w io.Writer, node *blackfriday.Node, entering
                lines := []string{}
                for _, line := range strings.Split(string(node.Literal), linebreak) {
                        trimmed := strings.Trim(line, " \t")
-                       if len(trimmed) == 0 {
-                               lines = append(lines, trimmed)
-                       } else {
-                               indented := r.Indentation + trimmed
-                               lines = append(lines, indented)
-                       } 
+                       indented := r.Indentation + trimmed
+                       lines = append(lines, indented)
                }
                
```
This is caused by an ignore-logic of an empty line like above. I forget to run the test with the original code.

Updated. Currently, the behavior is all the same in current test cases.